index.html

```diff
@@ -25,7 +25,9 @@
     <noscript><h3>You need JavaScript to follow the invitation.</h3></noscript>
     <h3 class="text-center" id="heading"></h3>
     <p class="text-center"><a class="btn btn-primary" id="button"></a></p>
-    <input type="url" class="form-control text-center" id="url_in" readonly></input>
```

Comment:
Looks like you are removing the </input> closing tag?

Reply:
It is redundant, https://www.jetbrains.com/help/phpstorm/html-redundant-closing-tag-for-html-element.html

Reply:
I personally don't like non-XML HTML, because you can't parse it with a normal XML parser. However, for pure HTML (not XHTML) it's the right thing to remove it here.

Reply:
I think what you intended to do was to replace `<input></input>` with `<input/>` but not with `<input>`. AFAIR, even in old-school HTML it's a good practice to close your tags (either way).

Reply:
I added `/>`